src/client/app/effects/github.ts

```diff
@@ -115,7 +115,7 @@ export class GitHubEffects {
             .map((gist: IGist) => ({ gist: gist, snippetId: id }))
             .catch(exception => {
                 if (!gistOwnerId && exception.status >= 400 && exception.status <= 499) {
-                    throw new Error(JSON.stringify({ id: id, gistOwnerId: UNKNOWN_GIST_OWNER_ID }));
+                    throw new PlaygroundError(JSON.stringify({ snippetId: id, gistOwnerId: UNKNOWN_GIST_OWNER_ID }));
```

Comment:
Should you add some sort of "type: UpdateGistFailed"?  So that you can then use it in your `.catch` block to check that this is exactly what you wanted (and not some other PlaygroundError that we eventually might add somewhere, which might be thrown from upstream?)